LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2020 Oleksiy Dubovyk
```

Comment:
Does is have to be me? Is it somehow legally required? Can it be 
`Copyright (c) 2020 Croton Team`
?